Dockerfile

```diff
@@ -15,6 +15,7 @@
 # Support FROM override
 ARG BUILD_IMAGE=docker.io/golang:1.24.7@sha256:5e9d14d681c3224276f0c8e318525ef6fc96b47fbcbb89f8bec0e402e18ea8bf
 ARG BASE_IMAGE=gcr.io/distroless/static:nonroot@sha256:9ecc53c269509f63c69a266168e4a687c7eb8c0cfd753bd8bfcaa4f58a90876f
+ARG LDFLAGS
```

Comment:
Hmm, building Dockerfile without Makefile leaves this empty, but the build result is exactly the same as with the old static flag. Might still be worth having the old static flag as the default here for being explicit.